.circleci/config.yml

```diff
@@ -9,40 +9,39 @@ jobs:
             - checkout
             - restore_cache:
                   keys:
-                      - v2-dependencies-{{ checksum "package.json" }}-{{ checksum "mui/package.json" }}-{{ checksum "rn-vector/package.json"}}
+                      - v2-dependencies-{{ checksum "package.json" }}-{{ checksum "packages/mui/package.json" }}-{{ checksum "packages/rn-vector/package.json"}}
             - run:
                   name: Install Dependencies
-                  command: |
-                      yarn install
-                      cd rn-vector && yarn install && cd ..
-                      cd mui && yarn install
+                  command: yarn install
```

Comment:
FYI, you'll want to make this `yarn install --immutable` so that no modifications are permitted to the lockfile (same as --frozen-lockfile in yarn 1)